### PR TITLE
fix(proxy): Add image proxy rate limit

### PR DESCRIPTION
**Pull request by ChristophWurst**
Avoid users sending too many requests through the server.

Ref https://docs.nextcloud.com/server/latest/developer_manual/basics/controllers.html#rate-limiting

### CODE REVIEW

**Review by kesselb — APPROVED** [review]
Make sense :+1:

**Comment by ChristophWurst** [comment]
Psalm needs to be silenced about the missing attribute for older versions

File: psalm.xml [hunk 1]
```diff
@@ -19,6 +19,11 @@
 		<directory name="lib/Vendor" />
     </extraFiles>
 	<issueHandlers>
+		<UndefinedAttributeClass>
+			<errorLevel type="info">
```
Comment (ChristophWurst) [inline psalm.xml:23]: I'm testing the consequences of using this over *suppress*

Comment (ChristophWurst) [reply psalm.xml:23]: Psalm passes :thinking: :tada: :thinking:

**Pull request merged**